.github/workflows/snapcraft_yaml.yml

```diff
@@ -1,25 +0,0 @@
-name: Has snapcraft.yaml been updated?
```

Comment:
This has been recently added (with the core24 PR). I suppose the removal is intentional? I have no strong opinions. I mean, such a check is nice and we could consider it non-blocking until we go a bit more official.

Reply:
It is intentional. I gave some details in the commit removing it. We now dynamically set the version on the snap at build time. So relying on changes on the `snapcraft.yaml` to make sure we incremented the version is not making sense anymore. I need to talk to other teams working on craft apps to understand how they manage to set a specific version/tag and then promote it. This is not our priority for this cycle but will need to be addressed before the first beta or candidate version of imagecraft.